src/controllers/bulk/bulksupported.h

```diff
@@ -1,18 +1,20 @@
-// A list of supported USB bulk devices
-
 #pragma once
 
-typedef struct bulk_supported {
-    unsigned short vendor_id;
-    unsigned short product_id;
-    unsigned char in_epaddr;
-    unsigned char out_epaddr;
-    unsigned int interface_number;
-} bulk_supported_t;
+#include <cstdint>
+
+// A list of supported USB bulk devices
+
+struct bulk_supported_t {
+    std::uint16_t vendor_id;
+    std::uint16_t product_id;
+    std::uint8_t in_epaddr;
+    std::uint8_t out_epaddr;
+    std::uint32_t interface_number;
```

Comment:
```suggestion
    std::uint8_t interface_number;
```
One byte is enough here: https://github.com/libusb/libusb/blob/467b6a8896daea3d104958bf0887312c5d14d150/libusb/libusb.h#L749

Reply:
done